trestle/core/remote/cache.py

```diff
@@ -110,11 +110,16 @@ def get_raw(self, force_update=False) -> Dict[str, Any]:
         self._update_cache(force_update)
         # Return results in the cache, whether yaml or json, or whatever is supported by fs.load_file().
         try:
-            return fs.load_file(self._cached_object_path)
-        except Exception as e:
-            logger.error(f'Cannot fs.load_file {self._cached_object_path}')
-            logger.debug(e)
-            raise TrestleError(f'Cache get failure for {self._uri}') from e
+            raw_data = fs.load_file(self._cached_object_path)
+        except Exception:
+            try:
+                # files from NIST seem to need cp1252 encoding
```

Comment:
NIST files are annotated with utf-8 as the extension type

Reply:
Oh - that comment should have been removed.  At first I thought NIST had an issue, but the issue was in cache.py where it wrote the file with pathlib.write_text().  That line now has encoding specified and before it did not - causing the problem.

Comment now removed.